src/sssom/util.py

```diff
@@ -156,17 +156,17 @@ def from_mapping_set_document(cls, doc: MappingSetDocument) -> "MappingSetDataFr
         df.replace("", np.nan, inplace=True)
         df.dropna(axis=1, how="all", inplace=True)  # remove columns with all row = 'None'-s.
 
+        slots = _get_sssom_schema_object().dict["slots"]
         slots_with_double_as_range = {
-            slot
-            for slot, slot_metadata in _get_sssom_schema_object().dict["slots"].items()
-            if slot_metadata["range"] == "double"
+            slot for slot, slot_metadata in slots.items() if slot_metadata["range"] == "double"
         }
         non_double_cols = df.loc[:, ~df.columns.isin(slots_with_double_as_range)]
-        non_double_cols = non_double_cols.replace(np.nan, "")
-        df[non_double_cols.columns] = non_double_cols
+        non_double_cols.replace(np.nan, "", inplace=True)
+        df.update(non_double_cols)
 
-        df = sort_df_rows_columns(df)
-        return cls.with_converter(df=df, converter=doc.converter, metadata=meta)
+        return cls.with_converter(
```

Comment:
why make this change that creates more diff and makes lines harder to read?

Reply:
It's compact and straightforward in my opinion.

Reply:
we can move purely aesthetic changes to a separate PR